extensions/meme/src/api/api.ts

```diff
@@ -0,0 +1,126 @@
+import fetch from 'node-fetch';
+import {ImgflipCaptionImageResponse, ImgflipGetMemesResponse} from './types';
+import {Meme} from 'types';
+import {getLocalStorageItem, setLocalStorageItem} from '@raycast/api';
+
+const IMGFLIP_USERNAME = 'raycast';
+const IMGFLIP_PASSWORD = 'aha_qkt1wnh2QGJ-hrv';
```

Comment:
Not sure if there's a way to encrypt credentials. The same credentials are used to login to the imgflip account and authenticate API requests. I created account this specifically for raycast.